api/accounts/accountUtil.ts

```diff
@@ -0,0 +1,29 @@
+import { FeeAccount } from "../interfaces/feeAccountPayload"
+import { http } from '../lib/http'
+
+export function getAccountUrl(baseUrl: string, accountName: string) {
+    return `${baseUrl}/accounts/${accountName}`
+}
+
+export function getAccount(accountNumber: string, url: string): Promise<any> {
+    return new Promise((resolve, reject) => {
```

Comment:
can we not just use async/await with a try/catch block?

Reply:
@paddybasi , I am using Promise.All here. If I change it now to use async/await that would mean changing lots of code which was already tested. We need to remember that this is not 1 feature. Many features which were added are now being added to develop from demo. Hence, I would prefer not to change any code until it's a sure stopper. Thanks.